week-2/debug/0.js

```diff
@@ -1,7 +1,9 @@
 // Predict and explain first...
 
 function multiply(a, b) {
-  console.log(a * b);
+  return a * b;
 }
 
-console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+console.log(`${multiply(10, 32)}`);
+
+//the multiply function didn't have a return statement and it returns undefined. So we need to add return to call the function.
```

Comment:
Your answer is right, you just need to improve the phrasing of your explanation. You've said "we need to add return to call the function" - this isn't quite right.